class/defaults.yml

```diff
@@ -13,3 +13,11 @@ parameters:
     token:
       timeoutSeconds: ~
       maxAgeSeconds: ~
+
+    ldapSync:
+      schedule: '*/5 * * * *'
```

Comment:
This is very short as a default. In cases there are 1000's of groups, this might exceed very fast. And in a best case this should be randomized to lower the load on the LDAP servers, when multiple syncs not starting at the same time.

Reply:
As discussed. Keep the 5min interval. Follow up ticket for randomization.

Reply:
See #4.